crypto/crypto_test.go

```diff
@@ -156,3 +176,18 @@ func createCrypto(t *testing.T) *Crypto {
 	}
 	return &c
 }
+
+func Test_validateKID(t *testing.T) {
+	t.Run("good KIDs", func(t *testing.T) {
+		assert.NoError(t, validateKID("admin-token-signing-key"))
+		assert.NoError(t, validateKID("did:nuts:2pgo54Z3ytC5EdjBicuJPe5gHyAsjF6rVio1FadSX74j#GxL7A5XNFr_tHcBW_fKCndGGko8DKa2ivPgJAGR0krA"))
+		assert.NoError(t, validateKID("did:nuts:3dGjPPeEuHsyNMgJwHkGX3HuJkEEnZ8H19qBqTaqLDbt#JwIR4Vct-EELNKeeB0BZ8Uff_rCZIrOhoiyp5LDFl68"))
+		assert.NoError(t, validateKID("did:nuts:BC5MtUzAncmfuGejPFGEgM2k8UfrKZVbbGyFeoG9JEEn#l2swLI0wus8gnzbI3sQaaiE7Yvv2qOUioaIZ8y_JZXs"))
+	})
+	t.Run("bad KIDs", func(t *testing.T) {
+		assert.Error(t, validateKID("../server-certificate"))
+		assert.Error(t, validateKID("\\"))
+		assert.Error(t, validateKID(""))
+		assert.Error(t, validateKID("\t"))
+	})
+}
```

Comment:
don't need to change this, but perfect usecase for table driven tests

Reply:
I think Go's data-driven test setup is quite unreadable and verbose, but that might just be me.

Reply:
I agree. This is the first time I spot a test where I think it makes sense